### PR TITLE
[MODULAR] More Taurs

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request

<details>
<summary> Taurs (No Minos, no Cens) </summary>

**Disclaimer,** these're heavily reliant on taurs, uh, not being broken and hiding the legs. So, none of these're gonna look _as_ good as they could for the time being. Gomen nasai.

image
image
image
_Alt Tentacle_ taur from _Virgo._ This should look substantially better, and also does wonders for your hips. 

image
image
image
_Chinese Dragon_ taur from _Virgo._ This one's just badass. C'mon. Look at the claws. Look at that tail. Wowza. Awooga.

image
image
_Wasp_ taur from _Virgo._ This one seems to scare people. I like that.

image
image
image
_Mantis_ taur (and arms) from _CitRP._ Imagine how good this would look if taurs were fixed. It looks cool, doesn't it.

image
image
image
_Slug_ taur from _Virgo._ You may be wondering why I didn't use a snailperson for this demonstration. Stop wondering.
</details>

## How This Contributes To The Skyrat Roleplay Experience
Adds new customization options borrowed from https://github.com/quotefox/Hyper-Station-13, https://github.com/VOREStation/VOREStation/pulls, https://github.com/Citadel-Station-13/Citadel-Station-13-RP, and https://github.com/sojourn-13/sojourn-station.

## Changelog

:cl:
add: More taur options, about five of them.
/:cl:

### CODE REVIEW

**Comment by rosextango** [comment]
Can we merge this one and the others? I'd really like to see them in the game and am very excited to use them.

**Comment by tf-4** [comment]
> Can we merge this one and the others? I'd really like to see them in the game and am very excited to use them.

patience, padawan.

**Review by tf-4 — APPROVED** [review]
Not sure how I feel on the sprites but I'm generally not really a fan of taurs so I won't comment on that. Code itself seems good.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm [hunk 1]
```diff
@@ -128,3 +128,33 @@
 	color_src = USE_ONE_COLOR
 	extra = TRUE
 	extra_color_src = MUTCOLORS2
+
+/datum/sprite_accessory/taur/chinesedragon
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm]:
```suggestion
/datum/sprite_accessory/taur/chinese_dragon
```

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm [hunk 2]
```diff
@@ -128,3 +128,33 @@
...
+	name = "Chinese Dragon"
+	icon_state = "noodle"
+	taur_mode = STYLE_TAUR_PAW
+
+/datum/sprite_accessory/taur/tentacle2
+	name = "Tentacle Alt"
+	icon_state = "tentaclealt"
+	taur_mode = STYLE_TAUR_SNAKE
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm:140]: Something tells me they're not meant to be treated like a naga tail.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm [hunk 3]
```diff
@@ -128,3 +128,33 @@
 	color_src = USE_ONE_COLOR
 	extra = TRUE
 	extra_color_src = MUTCOLORS2
+
+/datum/sprite_accessory/taur/chinesedragon
+	name = "Chinese Dragon"
+	icon_state = "noodle"
+	taur_mode = STYLE_TAUR_PAW
+
+/datum/sprite_accessory/taur/tentacle2
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm]:
```suggestion
/datum/sprite_accessory/taur/tentacle_alt
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Gonna put these on hold until we (read: I) get taur parts fixed, and for me to decide if I want to take taur stuff a step further. Because if I do, I don't know that all of these sprites will fit in-game anymore, since I've been thinking of auto-generating clothes a bit like the Teshari do it.

Aside from that, well... The squid-like one feels off, I can't quite put my finger on it, but there's something about it that feels like it's lacking heavily in details. Perhaps it's the shading or something.

The dragon one is just too big, that tail goes up *so* far. Really not a fan of it, at this point it's like using the megamoth mothwings (which I really think shouldn't be accessible, if they even are).

The wasp one... Is fine, I guess. It's hard to tell since you used colors on it.

The praying mantis shouldn't have the arms caked in (if anything, those should be wing options, but honestly I don't really think they're overly fitting in the aesthetic of the game, it's awkward to have a second pair of arms that just can't do anything).

The slug doesn't look great, but that's par for the course for slugs, I guess.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm (hunk 2 above)
Comment (Nerev4r) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm:140]: The other tentacle one is.

**Comment by Nerev4r** [comment]
> The praying mantis shouldn't have the arms caked in (if anything, those should be wing options, but honestly I don't really think they're overly fitting in the aesthetic of the game, it's awkward to have a second pair of arms that just can't do anything).

They aren't baked-in. The image was from pre-atomization when this was all bundled in one thing. I just used the mantis arms wings option with them for the purpose of the picture.

**Comment by Tokkihopp** [comment]
The slug, mantis, and wasp are all lovely :3

**Comment by tf-4** [comment]
Waiting on https://github.com/Skyrat-SS13/Skyrat-tg/pull/13848

**Comment by tf-4** [comment]
#13848 is merged

**Comment by Nerev4r** [comment]
gonna hit this another time

**Pull request closed without merging**